NitroxLauncher/Localization/Localization-EN.xaml

```diff
@@ -0,0 +1,80 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:local="clr-namespace:NitroxLauncher"
+                    xmlns:sys="clr-namespace:System;assembly=mscorlib">
+
+    <sys:String x:Key="LocalizationString">en</sys:String>
+    <sys:Int16 x:Key="Version">1</sys:Int16>
+    
+    <!-- Main Window -->
+    
+    <sys:String x:Key="NitroxMainTitle">Nitrox Launcher</sys:String>
+    <sys:String x:Key="NitroxTitle">NITROX</sys:String>
+    
+    <sys:String x:Key="NitroxPlayButton">PLAY GAME</sys:String>
+    <sys:String x:Key="NitroxPlayButtonTooltip">Play the game</sys:String>
+
+    <sys:String x:Key="NitroxServerButton">SERVER</sys:String>
+    <sys:String x:Key="NitroxServerButtonTooltip">Configure server options</sys:String>
+
+    <sys:String x:Key="NitroxOptionButton">OPTIONS</sys:String>
+    <sys:String x:Key="NitroxOptionButtonTooltip">Launcher settings</sys:String>
+
+    <sys:String x:Key="NitroxRelease">NITROX RELEASE</sys:String>
+    
+    <!-- Launch Game Page -->
+
+    <sys:String x:Key="NitroxGameTitle">Launch Game</sys:String>
+
+    <sys:String x:Key="NitroxPresentation" xml:space="preserve">
+    Nitrox is an open-source, multiplayer modification for the game Subnautica
+    
+    The mod is in the very early stages of development
+    However, a solid foundation has been laid for the community to iterate on
+    </sys:String>
+
+    <sys:String x:Key="NitroxGameSinglerplayerButton">SINGLEPLAYER</sys:String>
+    <sys:String x:Key="NitroxGameSinglerplayerButtonTooltip">Play alone</sys:String>
+
+    <sys:String x:Key="NitroxGameMultiplayerButton">MULTIPLAYER</sys:String>
+    <sys:String x:Key="NitroxGameMultiplayerButtonTooltip">Launch the game with multiplayer enabled</sys:String>
+
+    
+    <!-- Options Page -->
+
+    <sys:String x:Key="NitroxOptionTitle">Option Page</sys:String>
+    <sys:String x:Key="NitroxOTitle">Settings</sys:String>
```

Comment:
Let's not use odd abbreviations. Instead, give these both a proper name (including `NitroxOption` written in full) that explains the (usage) difference between the two.